prod/westeurope/eucovidcert/functions_eucovidcert/function_app/terragrunt.hcl

```diff
@@ -0,0 +1,107 @@
+dependency "resource_group" {
+  config_path = "../../resource_group"
+}
+
+# Subnet
+dependency "subnet" {
+  config_path = "../subnet"
+}
+
+dependency "subnet_appbackendl1" {
+  config_path = "../../../linux/appbackendl1/subnet"
+}
+
+dependency "subnet_appbackendl2" {
+  config_path = "../../../linux/appbackendl2/subnet"
+}
+
+
+# Common
+dependency "application_insights" {
+  config_path = "../../../common/application_insights"
+}
+
+dependency "key_vault" {
+  config_path = "../../../common/key_vault"
+}
+
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_function_app?ref=v3.0.3"
+}
+
+locals {
+  commonvars                   = read_terragrunt_config(find_in_parent_folders("commonvars.hcl"))
+  app_insights_ips_west_europe = local.commonvars.locals.app_insights_ips_west_europe
+  service_api_url              = local.commonvars.locals.service_api_url
+}
+
+inputs = {
+  name                = "eucovidcert"
+  resource_group_name = dependency.resource_group.outputs.resource_name
+
+  resources_prefix = {
+    function_app     = "fn3"
+    app_service_plan = "fn3"
+    storage_account  = "fn3"
+  }
+
+  app_service_plan_info = {
+    kind     = "elastic"
+    sku_tier = "ElasticPremium"
+    sku_size = "EP1"
+  }
+
+  runtime_version = "~3"
+
+  application_insights_instrumentation_key = dependency.application_insights.outputs.instrumentation_key
+
+  app_settings = {
+    FUNCTIONS_WORKER_RUNTIME       = "node"
+    WEBSITE_NODE_DEFAULT_VERSION   = "12.18.0"
```

Comment:
Is this node version a bit old, isn't it?   cc @balanza  (btw ... I've already applied the PR)

Reply:
I think we can go for v14.x, I see no problem and its a good opportunity to upgrade our runtimes. @gquadrati let's try locally if we can run it with v14, first @gquadrati

Reply:
@gquadrati can we go with node version 14.16.0. Is it required to update the "code" too? (cc @balanza )